BLSQ/iaso
# POLIO-951: Add a number field "Percentage of the target population to be covered" that will be encoded manually

Explain what problem this PR is resolving
- Add in risk assessment a number field "Percentage of the target population to be covered" that will be encoded manually

Related JIRA tickets : [POLIO-951](https://bluesquare.atlassian.net/browse/POLIO-951)

## Self proofreading checklist


## Changes
- Add percentage_covered_target_population in round model
- Add migration to create the field in the DB
- Add text input field in risk assessment form
- Define the field as a number in yup
- Add label translation

## How to test
- Go in campaigns
- Edit or create a campaign
- Open risk assessment tab
- Check if the percentage covered target population

## Print screen / video
![Peek 2023-04-12 17-37](https://user-images.githubusercontent.com/19631540/231509322-d069b6b8-c308-4728-a9a0-2cecfcb0096c.gif)



[POLIO-951]: https://bluesquare.atlassian.net/browse/POLIO-951?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

## Note
This PR will be merged into [POLIO-950-target-population-should-appear-on-top-of-doses-requested-field](https://github.com/BLSQ/iaso/tree/POLIO-950-target-population-should-appear-on-top-of-doses-requested-field)
